utility/src/main/java/org/oppia/android/util/parser/html/LiTagHandler.kt

```diff
@@ -5,6 +5,7 @@ import android.text.Editable
 import android.text.Spannable
 import android.text.Spanned
 import org.oppia.android.util.locale.OppiaLocale
+import org.oppia.android.util.parser.html.LiTagHandler.Companion.setSpanFromMark
```

Comment:
This import shouldn't be needed since the helper is part of the class's companion.

Reply:
Removed.